package.json

```diff
@@ -68,51 +68,41 @@
     ]
   },
   "dependencies": {
-    "@babel/polyfill": "7.12.1"
+    "chalk": "4.1.2",
+    "dotenv": "14.2.0",
+    "inquirer": "8.2.4",
+    "lodash": "4.17.21"
   },
   "devDependencies": {
-    "@swc-node/jest": "1.5.0",
-    "@testing-library/react": "11.2.7",
-    "@testing-library/react-hooks": "3.7.0",
-    "@testing-library/user-event": "13.5.0",
-    "axios-mock-adapter": "1.20.0",
+    "@swc-node/jest": "1.5.2",
     "babel-eslint": "10.1.0",
-    "chalk": "4.1.2",
-    "chokidar": "3.5.3",
-    "cross-env": "7.0.3",
-    "dotenv": "14.2.0",
+    "cross-env": "^7.0.3",
     "eslint": "7.32.0",
     "eslint-config-airbnb": "18.2.1",
-    "eslint-config-airbnb-base": "14.2.1",
     "eslint-config-prettier": "6.15.0",
     "eslint-plugin-import": "2.26.0",
     "eslint-plugin-jsdoc": "36.1.1",
     "eslint-plugin-jsx-a11y": "6.6.0",
     "eslint-plugin-node": "11.1.0",
     "eslint-plugin-react": "7.30.1",
     "eslint-plugin-react-hooks": "4.6.0",
-    "eslint-plugin-redux-saga": "1.3.2",
-    "execa": "1.0.0",
+    "execa": "5.1.1",
     "fs-extra": "10.1.0",
     "get-port": "5.1.1",
     "glob": "7.2.3",
     "husky": "3.1.0",
-    "istanbul": "~0.4.2",
     "jest": "28.1.2",
     "jest-circus": "28.1.2",
     "jest-cli": "28.1.2",
     "jest-environment-jsdom": "28.1.2",
     "jest-watch-typeahead": "0.6.5",
     "lerna": "5.1.6",
     "lint-staged": "10.5.4",
-    "lodash": "4.17.21",
-    "msw": "0.42.3",
     "npm-run-all": "4.1.5",
     "nx": "14.4.2",
     "plop": "2.7.6",
     "prettier": "1.19.1",
     "qs": "6.11.0",
-    "react-test-renderer": "17.0.2",
     "request": "2.88.2",
```

Comment:
I think we can remove `request` from the list of devDependencies here too ✨

Reply:
It would be really nice to remove request for good if possible: 
![image](https://user-images.githubusercontent.com/8593673/179076300-a092d6b0-be7b-4662-82e1-b57ae3d0eff0.png)

Reply:
`request` is required by `request-promise-native` which we are using :/